src/components/pacbio/PacbioLabwareSelectedList.vue

```diff
@@ -143,25 +143,29 @@ const pacbioPoolCreateStore = usePacbioPoolCreateStore() // A composable store f
 /**
  * A computed property that returns all selected requests
  * If the sortBySelection value is true, the requests are sorted by selection
- * @returns {Array} - An array of selected requests which can be sorted by selection
+ * @returns {Array} - An array of selected requests with its source labware which can be sorted by selection
  */
-const selectedRequests = computed(() => {
+const selectedRequestsWithSource = computed(() => {
   //get all selected requests first in the order of the labware scanned
-  const requests = props.labware.flatMap((labware) => {
-    if (isPlate(labware)) {
-      const plate = pacbioPoolCreateStore.selectedPlates.find(
-        (item) => item.barcode === labware.barcode,
-      )
-      return pacbioPoolCreateStore
-        .wellList(plate.wells)
-        .flatMap((well) => pacbioPoolCreateStore.requestList(well.requests || []))
-    } else {
-      const tube = pacbioPoolCreateStore.selectedTubes.find(
-        (item) => item.barcode === labware.barcode,
-      )
-      return pacbioPoolCreateStore.requestList(tube.requests || [])
-    }
-  })
+  const requests = props.labware
```

Comment:
Advisory Not needed for this cycle: Again I think there is oppurtunity for refactor here. There are a lot of similarities between the two paths so you could reduce it into a method with function passing.